package.json

```diff
@@ -47,24 +52,26 @@
     ]
   },
   "dependencies": {
-    "babel-plugin-transform-object-rest-spread": "^6.8",
-    "babel-preset-react": "^6.5",
-    "babel-preset-es2015": "^6.9"
+    "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-es2015": "^6.24.1"
   },
   "devDependencies": {
-    "assume": "^1.4",
+    "assume": "^1.5.1",
     "assume-sinon": "^1.0",
-    "babel-cli": "^6.18.0",
-    "babel-eslint": "^6.0",
-    "babel-register": "^6.9",
-    "godaddy-style": "^3.1.6",
-    "istanbul": "~1.0.0-alpha.2",
-    "jsdom": "^9.4.1",
-    "mocha": "^2.5",
+    "babel-cli": "^6.24.1",
+    "babel-register": "^6.24.1",
+    "eslint": "^3.19.0",
+    "eslint-config-godaddy-react": "^1.1.1",
+    "eslint-plugin-json": "^1.2.0",
+    "eslint-plugin-mocha": "^4.11.0",
+    "eslint-plugin-react": "^6.10.3",
+    "mocha": "^3.4.2",
+    "nyc": "^11.0.3",
+    "prop-types": "15.x.x",
```

Comment:
could we semver range this in exactly the same way the rest of the dependencies?